package.json

```diff
@@ -14,17 +14,24 @@
     "add-contributor": "all-contributors add",
     "generate-contributors": "all-contributors generate",
     "lint": "npm run lint:js & npm run lint:css",
-    "lint:js": "eslint .",
+    "lint:js": "eslint src",
```

Comment:
Do we need to lint everything, or just the source files?

Reply:
I'd still want to lint other dirs like example and website, and top level files such as the webpack configs. If eslint knows to only hit js files and to ignore the `dist`, this should work.